index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asad Shareef Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
```

Comment:
- [ ] I must commend your honest effort but it seems nothing has changed.

- Here is a snapshot of what your page looks like now: 

![image](https://user-images.githubusercontent.com/99715640/212410510-2daf6669-9d25-4785-a076-3d6cc30bcfb8.png)

- [ ] I think a major reason why your `background-image` is not showing is because you are trying to import an image file that you don't currently have in your `assets` folder.

- Check this 👇 
![image](https://user-images.githubusercontent.com/99715640/212411444-d82eb545-006f-452c-876b-c9cabae91d3f.png)
- [ ] Kindly check your file name
- [ ] Also, as the previous reviewer indicated, please check your `font` properties and `text-decoration` property on `LET'S CONNECT` element.